lib/controller/EetplanController.class.php

```diff
@@ -114,7 +114,15 @@ public function bekendehuizen($actie = null) {
             } else {
                 $this->view = $form;
             }
-
+        } elseif ($actie == 'verwijderen') {
+            $selection = filter_input(INPUT_POST, 'DataTableSelection', FILTER_SANITIZE_STRING, FILTER_FORCE_ARRAY);
+            $verwijderd = array();
+            foreach ($selection as $uuid) {
+                $eetplan = $this->model->getUUID($uuid);

```

Comment:
Je mag getUUID static laten om dit te mogen doen in PHP, zie GroepenController.

Als je uberhaupt van de static methode af wilt dan moet je ook even in ProfielModel refactoren, maar ik adviseer deze getters static te houden, het is tenslotte een convenience function.